Infrastructure.DataAccess/UserRepository.cs

```diff
@@ -44,5 +44,10 @@ public IQueryable<User> SearchOrganizationUsers(int organizationId, Maybe<string
                             .GetValueOrFallback(userQuery))
                     .Distinct();
         }
+
```

Comment:
I din UserService bruger du QueryByrightsHolder query, men hvis du nu flytter den viden herned dvs du laver en query. GetByOrganizationRole(role) som så bruger din QueryByOrganizationRole, så sørger dun USerService for at kalde dette repo som så med rette bruger query til at frembringe svaret.